README.md

```diff
@@ -40,3 +40,5 @@ When you're done working on the site, run the following command to move the post
 ```bash
 bundle exec rake integrate
 ```
+##
+[<img src="https://www.openhomefoundation.org/badges/home-assistant.png">](https://www.openhomefoundation.org/)
```

Comment:
We can avoid the use of HTML here, right?

Reply:
Ah yes we can now. It was markdown previously but we needed to force a specific size. I will amend